cmake/O2Dependencies.cmake

```diff
@@ -940,6 +940,7 @@ o2_define_bucket(
     its_reconstruction_bucket
     ITSReconstruction
     ITStracking
+    ITSSimulation
```

Comment:
The only reason why it did not depend before is that the Digit class was wrongly put into the ``ITSMFT/common/base``, which at some stage should be fixed.
Anyway, as I wrote you, I can single out the ROFrame length to separate option as ``AlpideParam.mROFrame`` (with the corresponding class in the base), but note that it still has to be set in the digitizer-workflow and picked up by the reco-flow.

Reply:
@shahor02  Potentially, the independence between the sim and rec libraries is still good,
even at the cost of digits being a part of the base.
The introduction of AlpideParam seems to be a  good idea.  This would allow us to maintain the "status quo".

Reply:
OK, I will move the ROFrame length to separate option, but the Digit.h should be actually moved to ````DataFormats/Detectors/ITSMFT/common/```` (as well as other Digits.h which were copy-pasted from the ITS...)

Reply:
@shahor02 Very good. Thank you, Ruben.
Also right:  Digit should be moved to DataFormats/  !   
Can this be done within this same PR ?   (Can be done later, of course. If you prefer.)